app/pages/about/publications-page.cjsx

```diff
@@ -152,7 +152,7 @@ publicationCategories =
       {citation: "Planet Hunters. VIII. Characterization of 41 Long-Period Exoplanet Candidates from Kepler Archival Data, Wang+ 2015."
       href: "http://arxiv.org/pdf/1512.02559v2.pdf"},
       {citation: "Planet Hunters. IX. KIC 8462852 – Where’s the flux?, Boyajian+ 2015."
-      href: "http://mnras.oxfordjournals.org/content/457/4/3988.full.pdf"},
+      href: "http://arxiv.org/pdf/1509.03622v1.pdf"},
```

Comment:
Any reason this is going back to the arXiv version, @mrniaboc? Where possible, we should try to link to the actual manuscript version if it's open access.